datastore/admin/datastore_admin_test.go

```diff
@@ -15,20 +15,26 @@
 package samples
 
 import (
+	"context"
 	"io/ioutil"
 	"testing"
 
 	"github.com/GoogleCloudPlatform/golang-samples/internal/testutil"
 )
 
-func TestCreate(t *testing.T) {
+func TestAdmin(t *testing.T) {
+	// Roles to be set in your Service Account and App Engine default service account
+	// to run this test:
+	// `Datastore Import Export Admin`, or `Cloud Datastore Owner`, or `Owner`,
+	// `Storage Admin`, or `Owner`.
```

Comment:
Thanks for adding this. Can we also link to the authority page here as well?

```suggestion
	// `Storage Admin`, or `Owner`.
	// See https://cloud.google.com/datastore/docs/export-import-entities#permissions for full details
```

Reply:
+1 to adding the link